spec/support/retry_helpers.rb

```diff
@@ -66,6 +66,8 @@ def reload_url_until_status_code(
     end
   end
 
+  alias reload_url_until_status_code expect_status_code_eventually
```

Comment:
I think these have to be the other way round

Reply:
Good spot, I've switched the order now.